src/AggregateFunctions/AggregateFunctionMinMaxAny.h

```diff
@@ -700,13 +700,11 @@ template <typename Data>
 class AggregateFunctionsSingleValue final : public IAggregateFunctionDataHelper<Data, AggregateFunctionsSingleValue<Data>>
 {
 private:
-    DataTypePtr type;
     SerializationPtr serialization;
```

Comment:
We can also get serialization if we need without storing as class member.

Reply:
It will be a virtual call in a loop.